modular_zubbers/code/modules/customization/sprite_accessories/genitals.dm

```diff
@@ -1,2 +1,186 @@
+	name = "Human"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/nondescript
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Nondescript"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/knotted
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Knotted"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/flared
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Flared"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/barbknot
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Barbed, Knotted"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/tapered
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Tapered"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/tentacle
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Tentacled"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/hemi
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Hemi"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/hemiknot
+	icon = 'modular_zubbers/icons/mob/sprite_accesory/genitals/penis_onmob.dmi'
+	name = "Knotted Hemi"
+	has_skintone_shading = TRUE
+
+/datum/sprite_accessory/genital/penis/human_old
+	icon_state = /datum/sprite_accessory/genital/penis/human::icon_state
+	name = /datum/sprite_accessory/genital/penis/human::name + " (Old)"
+	color_src = /datum/sprite_accessory/genital/penis/human::color_src
+	default_color = /datum/sprite_accessory/genital/penis/human::default_color
+	has_skintone_shading = /datum/sprite_accessory/genital/penis/human::has_skintone_shading
+	can_have_sheath = /datum/sprite_accessory/genital/penis/human::can_have_sheath
+
+/datum/sprite_accessory/genital/penis/nondescript_old
+	icon_state = /datum/sprite_accessory/genital/penis/nondescript::icon_state
+	name = /datum/sprite_accessory/genital/penis/nondescript::name + " (Old)"
+
+/datum/sprite_accessory/genital/penis/knotted_old
+	icon_state = /datum/sprite_accessory/genital/penis/knotted::icon_state
+	name = /datum/sprite_accessory/genital/penis/knotted::name + " (Old)"
+
+/datum/sprite_accessory/genital/penis/flared_old
+	icon_state = /datum/sprite_accessory/genital/penis/flared::icon_state
+	name = /datum/sprite_accessory/genital/penis/flared::name + " (Old)"
+
+/datum/sprite_accessory/genital/penis/barbknot_old
+	icon_state = /datum/sprite_accessory/genital/penis/barbknot::icon_state
+	name = /datum/sprite_accessory/genital/penis/barbknot::name + " (Old)"
+
+/datum/sprite_accessory/genital/penis/tapered_old
+	icon_state = /datum/sprite_accessory/genital/penis/tapered::icon_state
+	name = /datum/sprite_accessory/genital/penis/tapered::name + " (Old)"
```

Comment:
This pattern is going to fuck up people's preferences because they're stored in the save files by their names. Everyone with a character who had these set is going to have to manually switch back to the old sprites. 

Please undo this, and instead make the new sprites be "[old name] 2/alt/new/SPLURT/etc." instead.
That, or just ditch the old sprites entirely and make the new ones the only versions available.

Reply:
I'm looking into this, but if I attempt to have them as just separate entries as a whole, they simply don't show up when selected in the character customization screen in my localhost.